README.md

```diff
@@ -6,7 +6,6 @@
 
```

Comment:
Not sure about the changes in this file. Just a auto-formatting thing?

I'd remove the tweaks to this file.

Reply:
ya ignore this